docs/source/developer_guide/dependencies.rst

```diff
@@ -74,6 +74,13 @@ To add an estimator with a soft dependency, ensure the following:
    utility should be used in ``__init__``. Both the warning and constructor call should use the ``package_import_alias`` argument for this.
 *  If the soft dependencies require specific python versions, the ``python_version``
    tag should also be populated, with a PEP 440 compliant version specification ``str`` such as ``"<3.10"`` or ``">3.6,~=3.8"``.
+*  Skip the corresponding doctests.  To do this add a ``# doctest: +SKIP`` to the end of each line in the doctest to skip.
```

Comment:
Good point! I would formulate it from the position of not having examples yet rather than having examples and having to skip them.

E.g., sth like "if including docstring examples that use soft dependencies, ensure to skip doctest etc etc. To avoid reducing test coverage, consider exposing the example as a `pytest` test function instead, see below how to handle soft dependencies in `pytest` functions."